README.md

```diff
@@ -87,27 +82,40 @@ If you get the following message on Dokku, it probably means that you failed to
 Failed to connect to the frontend server... On Dokku, be sure that PRODUCTION is defined.  On localhost, open a second terminal window, cd into frontend and type: npm install; npm start";
 ```
 
-Additional environment variables that are needed may be found in `.env.SAMPLE` 
+Additional environment variables that are needed may be found in `.env.SAMPLE`
 
-In particular, you will need a value for the `UCSB_API_KEY`.  This is documented in `docs/ucsb_api_key.md`
+In particular, you will need a value for the `UCSB_API_KEY`. This is documented in `docs/ucsb_api_key.md`
 
 # Accessing swagger
 
 To access the swagger API endpoints, use:
 
-* <http://localhost:8080/swagger-ui/index.html>
-
+- <http://localhost:8080/swagger-ui/index.html>
 
 # To run React Storybook
 
-* cd into frontend
-* use: npm run storybook
-* This should put the storybook on http://localhost:6006
-* Additional stories are added under frontend/src/stories
+- cd into frontend
+- use: npm run storybook
+- This should put the storybook on http://localhost:6006
+- Additional stories are added under frontend/src/stories
 
-* For documentation on React Storybook, see: https://storybook.js.org/
+- For documentation on React Storybook, see: https://storybook.js.org/
 
 # Accessing Database Console
 
-* On localhost only: <http://localhost:8080/h2-console>  See also: [docs/h2-console.md](docs/h2-console.md)
-* On Dokku: see: <https://ucsb-cs156.github.io/topics/dokku/postgres_command_line.html>
+- On localhost only: <http://localhost:8080/h2-console> See also: [docs/h2-console.md](docs/h2-console.md)
+- On Dokku: see: <https://ucsb-cs156.github.io/topics/dokku/postgres_command_line.html>
+
+# Integration Testing
+
+To run a particular integration test (e.g. only `HomePageWebIT.java`) use `-Dit.test=ClassName`, for example:
+
+```
+INTEGRATION=true mvn test-compile failsafe:integration-test -Dit.test=HomePageWebIT
+```
+
+or to see it run live:
+
+```
+INTEGRATION=true HEADLESS=false mvn test-compile failsafe:integration-test -Dit.test=HomePageWebIT
+```
```

Comment:
Are there actually any integration tests in this code base yet?  If so, fine, but I'm skeptical.  Look and see.  If so, then you can keep this, but if not, you probably ought to remove it, since it's actually incorrect,